extensions.json

```diff
@@ -112,6 +112,14 @@
       "author": "Alexis Degrugillier",
       "url": "https://github.com/aledeg/FreshRSS-extensions",
       "type": "gh-subdirectory"
+    },
+    {
+      "name": "Hooktube",
+      "description": "Displays videos from YouTube feeds inline (fork of 'YouTube' Extension) but replaces every source by Hooktube",
```

Comment:
Remove the part `(fork of 'YouTube' Extension)` and probably add an explanation what makes it special, like `Displays videos from YouTube using the Hooktube service for enhanced privacy` or something like that, so users understand what is special about the extension.